pkg/domain/infra/abi/terminal/sigproxy_linux.go

```diff
@@ -16,6 +16,44 @@ import (
 // runc is using the same value.
 const signalBufferSize = 2048
 
+func ProxySignalsRemote(ctrID string, killFunc func(sig syscall.Signal) error) {
```

Comment:
I have no idea why this one is undefined on MacOS... @mheon any ideas?
Thanks.

Reply:
Because the file is named `_linux.go` therefore it will only compile on linux.

I recommend to move this function somewhere else, maybe `pkg/domain/infra/tunnel`? `pkg/domain/infra/abi` should only be used by local podman code and not the remote client.

Reply:
Yes that is correct, although we call podman-remote just podman on macos and windows.

Reply:
Thanks for your clarification!